en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-configuration.md

```diff
@@ -32,4 +32,6 @@ Run the following command to start the Kafka server:
 ```
 bin/kafka-server-start.sh config/server.properties
 ```
-Now that you have connected to Kafka, you can start publishing and consuming messages using the Kafka brokers. 
+Now that you have connected to Kafka, you can start publishing and consuming messages using the Kafka brokers.For more information, see
```

Comment:
```suggestion
Now that you have connected to Kafka, you can start publishing and consuming messages using the Kafka brokers. For more information, see
```

Shall we add the below links to this line itself?

Reply:
Done.